src/util/base.cc

```diff
@@ -104,18 +104,15 @@ bool process(
 
 bool BaseCommand::Impl(
     const std::string& cmd_name,
-    int argc,
```

Comment:
Why the move away from argc and argv?  I now cannot parse out -a as an option for benchmark .  It thinks it is -all for kovri-util, when i run kovri-util benchark -a the help prints out for me and the -all is NOT run anyway.